Week3/HW3/3_moreJavascript/15_testEquality.js

```diff
@@ -0,0 +1,36 @@
+// x == y it should be true since it is not strict equality.
+// x === y  it should be false since it is strict equality.
+// z == y  that is true since it is defined like that.
+// z == x it should be true since it is not strict equality.
+
+
+function same(a,b){
+    var same = a.length === b.length,
```

Comment:
Just wondering if you used an example from the Internet to think about this logic? 😉